internal/config/config.go

```diff
@@ -28,7 +28,7 @@ func Read(path string) (config types.Configuration, err error) {
 			config.Remotes[i].Auth.AccessToken = string(content)
 		}
 		if remote.Timeout == 0 {
-			remote.Timeout = 30
+			remote.Timeout = 300
```

Comment:
I noticed a merge conflict with the prometheus PR when testing this PR:

```suggestion
			config.Remotes[i].Timeout = 300
```

Reply:
Thanks. (This also explain why the test was not failing :/)